### PR TITLE
Update mouse events

### PR DESCRIPTION
**Pull request by irenesmith**
Based on Sheppy's comments in [MDN/Sprints #955](https://app.zenhub.com/workspaces/mdn-sprint-board-5ab3d23fd0f4ea53b0022a61/issues/mdn/sprints/955), I've updated the BCD for mouse events to include missing information.

### CODE REVIEW

File: api/Element.json [hunk 1]
```diff
@@ -4529,6 +4578,55 @@
...
+      "MozMousePixelScroll_event": {
+        "__compat": {
+          "mdn_url": "https://developer.mozilla.org/docs/Web/API/Element/MozMousePixelScroll_event",
+          "description": "<code>MozMousePixelScroll</code> event",
+          "support": {
+            "chrome": {
+              "version_added": false
+            },
+            "chrome_android": {
+              "version_added": true
```
Comment (jpmedley) [inline api/Element.json]: This should be false.

File: api/Element.json [hunk 2]
```diff
@@ -7845,6 +7943,178 @@
...
+              "version_added": false
+            },
+            "opera_android": {
+              "version_added": false
+            },
+            "safari": {
+              "version_added": true
+            },
+            "safari_ios": {
+              "version_added": true
```
Comment (jpmedley) [inline api/Element.json]: `"webview_android"` is missing from this and several others. It should match `"chrome"` (at least for this API). Please add it throughout.

File: api/Element.json (hunk 1 above)
Comment (a2sheppy) [reply api/Element.json]: This should definitely be fixed before this PR is merged.

File: api/Element.json (hunk 2 above)
Comment (a2sheppy) [reply api/Element.json]: IMO this should be done in a follow-up PR.

Comment (irenesmith) [reply api/Element.json]: I have made the update and pushed it.

**Comment by chrisdavidmills** [comment]
Joe has approved, so merging.

**Pull request merged**